Uppsala/UppEt41.xml

```diff
@@ -0,0 +1,501 @@
+                                    </q>
+                                </item>
+                                
+                                <item xml:id="e8">
+                                    <desc type="AcquisitionNote">Note in pen on the notecard listed as <ref target="#e7"/>.</desc>
+                                    <q xml:lang="sv">
+                                        x 1968/31
+                                    </q>
+                                </item>
+                                
+                            </list>
+                        </additions>
+
+                        <bindingDesc>
+                            <binding xml:id="binding">
+                                <decoNote xml:id="b1" type="Boards">European binding with quarter-leather cover.</decoNote>
+                            </binding>
+                        </bindingDesc>
+                        
+                    </physDesc>
+                                        
+                    <history>
+                        <origin>
+                            <origDate when="1605">1605</origDate>
+                            According to a note on <locus target="#218vb"/>, the manuscript is dated to
+                            <date when="1705">1705</date> "according to the calculation of the Europeans"
+                            (<foreign xml:lang="gez"><supplied reason="omitted" resp="PRS6337Lofgren">በ</supplied>ሐሳቦሙ፡ ለአፍርንጅ፡</foreign>)
+                            and to <date calendar="ethiopian" when-custom="1597">1597</date> "according to the calculation of the Copts"
+                            (<foreign xml:lang="gez">ወበሐሳበ፡ ቅብጣውያንሰ፡ <gap reason="ellipsis"/></foreign>).
+                            <bibl><ptr target="bm:Loefgren1974Katalog"/><citedRange unit="page">130</citedRange></bibl> 
+                            suggests an emendation of 
+                            <date when="1705">1705</date> to <date when="1605">1605</date>, which would bring the
+                            date into agreement with the Coptic/Ethiopian year.
+                        </origin>
+                        
+                        <provenance>
+                            In <date when="1929">1929</date>, when the manuscript was briefly described in 
+                            <bibl><ptr target="bm:Loefgren1929Evangeliska"/><citedRange unit="page">20</citedRange>
+                                <citedRange unit="number">Nachtrag 7</citedRange></bibl> based on information provided by the current
+                            head of the Swedish Mission Station in <placeName ref="LOC1579Asmara"/>
+                            Jonas Iwarson, the manuscript was located in the library of the Mission.
+                        </provenance>
+                        
+                        <acquisition>According to <ref target="#e8">, the manuscript reached its current repository in 
```

Comment:
I think`<ref target="#e8">` should be closed and `</ref>` at the end of the sentence removed.